lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,19 +125,25 @@ public boolean isConnected() {
 
   @Override public Location getLastLocation(LostApiClient client) {
     throwIfNotConnected(client);
+    Location location = null;
     try {
-      return service.getLastLocation();
+      location = service.getLastLocation();
     } catch (RemoteException e) {
-      throw new RuntimeException(e);
+      Log.e(TAG, "Error occurred trying to get last Location " + e.getStackTrace());
+    } finally {
+      return location;
     }
   }
 
   @Override public LocationAvailability getLocationAvailability(LostApiClient client) {
     throwIfNotConnected(client);
+    LocationAvailability availability = null;
     try {
-      return service.getLocationAvailability();
+      availability = service.getLocationAvailability();
     } catch (RemoteException e) {
-      throw new RuntimeException(e);
+      Log.e(TAG, "Error occurred trying to get LocationAvailability " + e.getStackTrace());
+    } finally {
+      return availability;
```

Comment:
Will return null on error. Assuming this how Google's APIs behave as well, it would be good to mention this in the javadoc.